src/frontend/screens/Observation/Buttons.tsx

```diff
@@ -8,16 +8,15 @@ import {useNavigationFromRoot} from '../../hooks/useNavigationWithTypes';
 import {useDeleteObservation} from '../../hooks/server/observations';
 import {Text} from '../../sharedComponents/Text';
```

Comment:
can you replace this Text Element?

Reply:
[9af300f](https://github.com/digidem/comapeo-mobile/pull/847/commits/9af300f79d54f28a85d7487d12fb983f71421c2f)